.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }} [ ${{ matrix.os }} ]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Since all these items are are explicitly given in each entry, do these lines do anything? If not, I'd remove them.

Reply:
That line allows to run base tests (with  python: [3.8] and toxenv: [test]) on all platform:
first 3 items in https://github.com/avalentino/pyerfa/actions/runs/363098156.
Than additional configuration are "include"d into the matrix for oldestdeps, devdeps, codestyle and build_docs.

Reply:
Ah, that makes sense! Thanks.